webservices/cron.php

```diff
@@ -320,12 +318,12 @@ function CronExec($oP, $bVerbose, $bDebug=false)
 				}
 			}
 		}
-
-		if ($bVerbose && $bWorkDone)
-		{
-			$oP->p("Sleeping...\n");
+		if ($aTasks == []) {
```

Comment:
Wouldn't it be better to do this?
```suggestion
		if (empty($aTasks)) {
```

Reply:
Yes, but we rather use `count($aTasks) === 0` than `empty()`